data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/task/ImportTask.java

```diff
@@ -73,7 +73,7 @@ public ImportTaskResult execute() {
       return ImportTaskResult.builder()
           .rawRecord(params.getSourceRecord())
           .rowNumber(params.getRowNumber())
-          .targets(Collections.singletonList(singleTargetResult))
+          .targets(new ArrayList<>(Collections.singletonList(singleTargetResult)))
```

Comment:
What is the motivation behind this change?

Reply:
@Torch3333 
Earlier I used the `Collections.singletonList(singleTargetResult)` which caused an exception(java.lang.UnsupportedOperationException) when trying to update the value (which is done in case of an exception) as it be immutable. So to fix this issue I wrapped it with `ArrayList<>()`.